packages/artillery/lib/platform/aws-ecs/worker/Dockerfile

```diff
@@ -2,34 +2,57 @@
 # NOTE: Version we use here needs to be kept consistent with that in
 # artillery-engine-playwright.
 # ********************************
-FROM mcr.microsoft.com/playwright:v1.43.1
-LABEL maintainer="team@artillery.io"
+FROM mcr.microsoft.com/playwright:v1.43.1 as build-image
+
+# https://steele.blue/playwright-on-lambda/
```

Comment:
What's the relevance of these URLs? Can we add relevant comments inline if appropriate instead?

Reply:
these were more for getting Playwright to work with Lambda. I'll keep them off for now, since I will address that separately in the end, and add it back if necessary